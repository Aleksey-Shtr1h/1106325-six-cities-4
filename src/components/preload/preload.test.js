import React from 'react';
import renderer from 'react-test-renderer';

import {Preload} from './preload.jsx';


it(`Should Preload render correctly`, () => {

  const tree = renderer.create(
      <Preload
      />,
      {
        createNodeMock: () => {
          return document.createElement(`div`);
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
