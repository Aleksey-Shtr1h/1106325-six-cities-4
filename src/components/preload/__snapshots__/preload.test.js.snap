// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Preload render correctly 1`] = `
<div
  className="loading-network"
>
  <h1
    className="loading-network__title"
  >
    Загрузка...
  </h1>
</div>
`;
